### ISSUE TITLE
In multiplayer `WorldService.GetMarkPanels` errors in lua when Game Master role is in use

### ISSUE THREAD

**Issue by bitbubbler** [DCS-gRPC/rust-server#156]
steps to reproduce:
- start a blank miz file with dcs-grpc, include at least one game master slot
- connect to the multiplayer, join the game master slot
- place a marker using the game master role
- make an rpc call for `WorldService.GetMarkPanels`
- expect lua error below

error from the logs:
```2022-06-04 22:42:46.068 ERROR   dcs_grpc: error executing getMarkPanels: [string "<redacted>\Saved Games\DCS\Scripts\DCS-gRPC\exporters\object.lua"]:28: attempt to index local 'unit' (a nil value)
stack traceback:
	[string "<redacted>\Saved Games\DCS\Scripts\DCS-gRPC\exporters\object.lua"]:28: in function 'unit'
	[string "<redacted>\Saved Games\DCS\Scripts\DCS-gRPC\exporters\object.lua"]:122: in function 'markPanel'
	[string "<redacted>\Saved Games\DCS\Scripts\DCS-gRPC\methods\world.lua"]:36: in function <[string "<redacted>\Saved Games\DCS\Scripts\DCS-gRPC\methods\world.lua"]:31>
	(tail call): ?
	[C]: in function 'xpcall'
	[string "<redacted>\Saved Games\DCS\Scripts\DCS-gRPC\grpc.lua"]:167: in function <[string "<redacted>\Saved Games\DCS\Scripts\DCS-gRPC\grpc.lua"]:163>
	[C]: in function 'next'
	[string "<redacted>\Saved Games\DCS\Scripts\DCS-gRPC\grpc.lua"]:200: in function <[string "<redacted>\Saved Games\DCS\Scripts\DCS-gRPC\grpc.lua"]:198>
	[C]: in function 'pcall'
	[string "<redacted>\Saved Games\DCS\Scripts\DCS-gRPC\grpc.lua"]:211: in function <[string "<redacted>\Saved Games\DCS\Scripts\DCS-gRPC\grpc.lua"]:209>
2022-06-04 22:42:46.069 ERROR   dcs_grpc: Error retrieving next command: attempt to concatenate a userdata value```

**Comment by rkusa** [comment]
Thanks a lot for the report and especially for writing down steps to reproduce 👍. The error will be fixed by #157

**Comment by bitbubbler** [comment]
For future readers: Through research it seem that this also impacts jtac and tac command roles in addition to game master roles
